python/flask/app/appsignal_config.py

```diff
@@ -0,0 +1,8 @@
+from appsignal import Appsignal
+
+appsignal = Appsignal(
+    active=True,
+    name="python/flask",
+    environment="development",
+    log_level="trace",
```

Comment:
No need to specify `log_level="trace"` here, this is already done in the `appsignal.env` file [at the root of this repository](https://github.com/appsignal/test-setups/blob/main/appsignal.env#L4).
```suggestion
```